iiif_prezi3/helpers/make_manifest.py

```diff
@@ -0,0 +1,20 @@
+from ..loader import monkeypatch_schema
+from ..skeleton import Collection, Manifest
+
+
+class MakeManifest:
+
+    def make_manifest(self, **kwargs):
+        """
+        Creates a new Manifest, adds a Reference to it to the 
+        calling Collection items and returns the newly created Manifest.
+        Accepts keyword arguments to customize the resulting instance.
+        """
+
+        manifest = Manifest(**kwargs)
+        reference = manifest.to_reference()
+        self.add_item(reference)
```

Comment:
`add_item` now handles doing the reference conversion, so you should be able to replace this with `self.add_item(manifest)